### ISSUE TITLE
Doku: Rework the notes section to not show default related links in all cmdlets

### ISSUE THREAD

**Issue by SebastianSchuetze** [DarqueWarrior/vsteam#436]
_Content of the issue has been copied from PR #418 by @smholvoet_

------

Another suggestion (which I have not yet pushed to my branch) is reworking the Notes section. I personally think it's very annoying seeing relevant cmdlets mentioned there, which are not clickable. Granted, they are mentioned a bit further down in the Related links section, but I think making the cmdlets clickable in the Notes section prevents a user from having to scroll further down and find the exact cmdlet he was looking for in a list of (sometimes) many more cmdlets:

![image](https://user-images.githubusercontent.com/7256168/149033790-fc2e48c7-1be1-4842-b1be-7f2d418d3409.png)

Noticed some cmdlets have related links which get added below the cmdlets which are displayed through the prerequisites.md file, for example Update-VSTeamPolicy, which I think is particularly useful.

In summary:

Remove  from the Related links section
Rework the Notes section to include link, making things easier for users
